samples/v1beta1/requirements.txt

```diff
@@ -0,0 +1 @@
+google-cloud-datacatalog==3.0.0
```

Comment:
should this be a hard pin, or will it get updated when submitted by a bot?  We're at 3.4.0 now from what I see..

Reply:
Changed to >=3.4.0